packages/coin98/src/adapter.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  EventEmitter,
+  pollUntilReady,
+  WalletAccountError,
+  WalletAdapter,
+  WalletAdapterEvents,
+  WalletNotConnectedError,
+  WalletNotFoundError,
+  WalletPublicKeyError,
+  WalletSignatureError,
+  WalletNotInstalledError
+} from '@solana/wallet-adapter-base';
+import { PublicKey, Transaction } from '@solana/web3.js';
+import bs58 from 'bs58'
```

Comment:
I see there's a peer/dev dependency on `@types/bs58`, but this suggests that there should be a dependency on both `bs58` and `@types/bs58`, and remove the peer and dev deps (we publish both compiled `lib` and uncompiled TS `src` dirs to npm)

Reply:
fixed.